src/Provider/GoogleMaps/Model/GoogleAddress.php

```diff
@@ -118,4 +123,39 @@ public function withSubpremise(string $subpremise = null)
 
         return $new;
     }
+
+    /**
+     * @return array SubLocalityLevelCollection
+     */
+    public function getSubLocalityLevels(): SubLocalityLevelCollection
```

Comment:
You cannot garuantee this return value unless you initialize this property in the constructor.